dotnet/src/SemanticKernel.Skills/Skills.Memory.Sqlite/Database.cs

```diff
@@ -43,6 +44,42 @@ public static async Task<SqliteConnection> CreateConnectionAsync(string filename
         await cmd.ExecuteNonQueryAsync(cancel);
     }
 
+    public static async Task InsertOrIgnoreAsync(this SqliteConnection conn,
+    string collection, string key, string? value, string? timestamp, CancellationToken cancel = default)
```

Comment:
spacing - line wraps should be spaced/tabbed in. Visual Studio should have the ability to autoformat documents to fix these formatting issues.